tests/test_opensearch.py

```diff
@@ -0,0 +1,250 @@
+import logging
+
+import boto3
+import pandas as pd
+import json
+
+import awswrangler as wr
+
+
+logging.getLogger("awswrangler").setLevel(logging.DEBUG)
+
+# TODO: create test_infra for opensearch
+OPENSEARCH_DOMAIN = 'search-es71-public-z63iyqxccc4ungar5vx45xwgfi.us-east-1.es.amazonaws.com'  # change to your domain
```

Comment:
We usually use pytest fixtures that fetch the values from CloudFormation outputs, [example](https://github.com/awslabs/aws-data-wrangler/blob/main/tests/conftest.py#L27)